templates/products/products-listing-row.php

```diff
@@ -230,6 +230,15 @@
         } else {
             echo '<span class="na">&ndash;</span>';
         }
+        /**
+         * Fire an action to add extra content after earning column in product listing table
+         *
+         * @since 3.14.11
+         *
+         * @param \WC_Product $product Current product.
+         * @param float $earning Current product's earning.
+         */
+        do_action( 'dokan_product_list_table_after_column_content_earning', $product, $earning );
```

Comment:
- `@since 3.14.11` should be  `@since DOKAN_SINCE`. `DOKAN_SINCE` will be replaced with associated released version.